src/datepicker/datepicker-input.ts

```diff
@@ -494,7 +496,9 @@ export class NgbInputDatepicker implements OnChanges,
 
   ngOnDestroy() {
     this.close();
-    this._zoneSubscription.unsubscribe();
+    if (this._zoneSubscription) {
```

Comment:
I would use `this._zoneSubscription?.unsubscribe();` instead of these three lines (same for ngOnDestroy)

Reply:
Fixed